docs/project/changelog.md

```diff
@@ -26,6 +26,26 @@ myst:
   compile time and link time.
   {pr}`5320`
 
+### `python` CLI entrypoint
```

Comment:
I think many users who don't know about `pyodide venv` would be confused about what we're referring to when we say `python CLI entrypoint`. How about making the title a little clearer (here and in 0.27.4 section)

Reply:
Do you have a specific suggestion?